docs/install.md

```diff
@@ -64,7 +66,20 @@ conda install pytorch=1.3.1 cudatoolkit=9.2 torchvision=0.4.2 -c pytorch
 
 If you build PyTorch from source instead of installing the prebuilt package, you can use more CUDA versions such as 9.0.
 
-c. Install mmcv, we recommend you to install the pre-build mmcv as below.
+## Install MMPose
```

Comment:
A little suggestion about this part:
1. Maybe add level-3 titles for "Install MMPose via MIM" and "Install MMPose manually"
2. Installing mmpose by "mim install mmpose" is basically equivalent to "pip install mmpose", which would have a different result from the manual installation (`git clone + pip install -e .`). Users who wish to install mmpose in development mode via mim need to run `git clone + mim install -e .`. I think we should make it clear. Perhaps we need to reorganize this part to distinguish different installation cases: 1) mim/not mim; 2) develop mode/not develop mode

Reply:
I have removed this part, because mim is not ready yet.